### ISSUE TITLE
Please push wyng-backup as default backup tool for qubes-os!

### ISSUE THREAD

**Issue by gangamstyle** [tasket/wyng-backup#102]
Hello, I am using wyng-backup for over a year and I must say that I made decision to use qubes-os as my main OS only after I discovered your awesome backup tool.
Please push it into qubes, because right now qubes has very bad and inefficient backup system.
Thank you!

**Comment by tasket** [comment]
BTW this has a Qubes issue: https://github.com/QubesOS/qubes-issues/issues/858

**Comment by tasket** [comment]
Minimum ToDo list I think is needed for Qubes acceptance (subject to change):

- [ ] Working VM settings integration script
- [ ] V0.4 beta or full release

Qubes OS ToDo:
- [ ] Separation of .cache into separate volume, or a way to delete .cache on VM exit

---

Nice to have for Qubes:
- [ ] BTRFS support in Wyng

**Comment by tlaurion** [comment]
> Minimum ToDo list I think is needed for Qubes acceptance (subject to change):
> 
> - [ ] Working VM settings integration script
> - [ ] V0.4 beta or full release
> 
> Qubes OS ToDo:
> - [ ] Separation of .cache into separate volume, or a way to delete .cache on VM exit
> 
> ---
> 
> Nice to have for Qubes:
> - [ ] BTRFS support in Wyng

@tasket wyng-util-qubes fixes first point. This needs a quick update as well.
